lib/p2p/Pool.ts

```diff
@@ -728,7 +728,11 @@ class Pool extends EventEmitter {
     const orderInvalidationPacket = new packets.OrderInvalidationPacket({ id, pairId, quantity });
     this.peers.forEach(async (peer) => {
       if (!nodeToExclude || peer.nodePubKey !== nodeToExclude) {
-        await peer.sendPacket(orderInvalidationPacket);
+        if (peer.advertisedPairs.includes(pairId)) {
+          await peer.sendPacket(orderInvalidationPacket);
+        } else {
+          this.logger.trace(`skipping to send order invalidation to ${peer.address} since pair is not found`);
```

Comment:
I don't think we need this logging statement even on the trace level, it's basically a non-event.